tarok/src/cards.cpp

```diff
@@ -117,24 +117,11 @@ void Shuffle(std::vector<open_spiel::Action>* actions, std::mt19937&& rng) {
 
 int CardPoints(const std::vector<open_spiel::Action>& actions,
                const std::array<Card, 54>& deck) {
-  int points = 0;
-  // count cards in batches of three
-  for (int i = 0; i < actions.size() / 3 * 3; i += 3) {
-    points +=
-        (deck.at(actions.at(i)).points + deck.at(actions.at(i + 1)).points +
-         deck.at(actions.at(i + 2)).points) -
-        2;
+  float points = 0;
+  for (int i = 0; i < actions.size(); i++) {
+    points += static_cast<float>(deck.at(actions.at(i)).points) - 0.666;
```

Comment:
This could simply be `points += deck.at(actions.at(i)).points - 0.666f;` methinks.

Reply:
Did some minor refactoring.